python/cugraph/cugraph/tests/test_property_graph.py

```diff
@@ -1845,6 +1845,27 @@ def test_add_data_noncontiguous(df_type):
         )
 
 
+@pytest.mark.parametrize("df_type", df_types, ids=df_type_id)
+def test_get_num_vertices(df_type):
```

Comment:
Rather than add another test with a very similar name, can `test_num_vertices()` be updated to add this coverage? Otherwise, it's not clear how the intent of `test_get_num_vertices()` differs from `test_num_vertices()`.

(I like this new name better since it matches the actual API name, so if you combine them my vote would be to use the new name)

Reply:
Thanks for the careful review.  Good catch.  My bad.

I changed the name and added a description.  It tests a very specific case.